system-variables.md

```diff
@@ -1153,18 +1153,18 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 </CustomContent>
 
-### tidb_ddl_distribute_reorg <span class="version-mark">New in v6.6.0</span>
+### tidb_enable_dist_task <span class="version-mark">New in v7.1.0</span>
 
 > **Warning:**
 >
 > - This feature is still in the experimental stage. It is not recommended to enable this feature in production environments.
-> - When this feature is enabled, TiDB only performs simple retries when an exception occurs during the DDL reorg phase. There is currently no retry method that is compatible with DDL operations. That is, you cannot control the number of retries using [`tidb_ddl_error_count_limit`](#tidb_ddl_error_count_limit).
 
 - Scope: GLOBAL
 - Persists to cluster: Yes
 - Default value: `OFF`
-- This variable is used to control whether to enable distributed execution of the DDL reorg phase to improve the speed of this phase. Currently, this variable is only valid for the `ADD INDEX` statement. Enabling this variable improves the performance of large tables. Distributed DDL execution can control the CPU usage of DDL through dynamic DDL resource management to prevent DDL from affecting the online application.
-- To verify whether a completed `ADD INDEX` operation is accelerated by this feature, you can check whether a corresponding task is in the `mysql.tidb_background_subtask_history` table.
+- This variable is used to control whether to enable [the distributed execution framework](/tidb-distributed-execution-framework.md). After enabling distributed execution, backend tasks such as DDL and import will be jointly completed by multiple TiDB nodes in the cluster.
```

Comment:
As tidb-distributed-execution-framework.md will be ready next month, the reference to it can be removed for now.